platform/android/display_server_android.cpp

```diff
@@ -659,17 +664,75 @@ void DisplayServerAndroid::process_hover(int p_type, Point2 p_pos) {
 	}
 }
 
-void DisplayServerAndroid::process_double_tap(Point2 p_pos) {
+void DisplayServerAndroid::process_mouse_event(int p_action, int p_button_mask, Point2 p_pos, float factor) {
+	switch (p_action) {
+		case AMOTION_EVENT_ACTION_BUTTON_PRESS:
+		case AMOTION_EVENT_ACTION_BUTTON_RELEASE: {
+			Ref<InputEventMouseButton> ev;
+			ev.instance();
+			ev->set_position(p_pos);
+			ev->set_global_position(p_pos);
+			ev->set_pressed(p_action == AMOTION_EVENT_ACTION_BUTTON_PRESS);
+			ev->set_factor(factor);
+			int button_mask = buttons_state ^ p_button_mask;
+
+			buttons_state = p_button_mask;
+
+			ev->set_button_index(button_index_from_mask(button_mask));
+			ev->set_button_mask(buttons_state);
+			Input::get_singleton()->parse_input_event(ev);
+		} break;
+
+		case AMOTION_EVENT_ACTION_MOVE: {
+			Ref<InputEventMouseMotion> ev;
+			ev.instance();
+			ev->set_position(p_pos);
+			ev->set_global_position(p_pos);
+			ev->set_relative(p_pos - hover_prev_pos);
+			ev->set_button_mask(p_button_mask);
+			Input::get_singleton()->parse_input_event(ev);
```

Comment:
Other platforms use `accumulate_input_event` and `flush_accumulated_events` instead of directly calling `parse_input_event`, which is used to report motion only once per frame when accumulated events are enabled (it's the default in the editor). I'm not sure if it's something to worry about for this PR, but it might be needed for editor support on android in order to make things consistent with other platforms.

Reply:
Yeah, I saw that, it's a bit more complicated than just using that, as it needs some configuration in os_android (if I'm not wrong)

I would probably change that in a different PR when I'll have the time